Game/Scripts/PlayerRotationScript.cpp

```diff
@@ -0,0 +1,53 @@
+#include "PlayerRotationScript.h"
+
+#include "Application.h"
+#include "ModuleInput.h"
+
+#include "Components/ComponentScript.h"
+#include "Components/ComponentMeshCollider.h"
+#include "Components/ComponentRigidBody.h"
+#include "Components/ComponentTransform.h"
+
+REGISTERCLASS(PlayerRotationScript);
+
+PlayerRotationScript::PlayerRotationScript() : Script(), componentTransform(nullptr)
+{
+}
+
+void PlayerRotationScript::Start()
+{
+	componentTransform = static_cast<ComponentTransform*>(owner->GetComponent(ComponentType::TRANSFORM));
+}
+
+void PlayerRotationScript::PreUpdate(float deltaTime)
+{
+	Rotate();
+}
+
+void PlayerRotationScript::Rotate()
+{
+	float deltaTime = App->GetDeltaTime();
+	ComponentTransform* trans = static_cast<ComponentTransform*>(owner->GetComponent(ComponentType::TRANSFORM));
+	float3 newRot = trans->GetRotationXYZ();
+	ModuleInput* input = App->GetModule<ModuleInput>();
+	newRot.y += -input->GetMouseMotion().x * deltaTime;
+	trans->SetRotation(newRot);
+	trans->UpdateTransformMatrices();
+
+
+	//Corroborate that you don't fuse with a owner
```

Comment:
What do you mean with "fuse"?
```suggestion
	//Corroborate that you don't fuse with an owner
```